docs/cloud-protocol/DeviceDataAndConfiguration.rst

```diff
@@ -7,6 +7,13 @@ Device data and configuration
    :local:
    :depth: 2
 
+The schemas referenced on this page define the data format used by the device to encode data.
+Currently, :ref:`JSON <json-format>` is the only supported encoding format.
+
+While the schemas mark properties as required, the device can always send partial updates.
+For example, the device might initially report the full roaming information after boot, but later only send the changed values (typically RSRP) to the cloud to conserve data and power.
```

Comment:
What about the abbreviation RSRP? Would it be worth opening?

Reply:
It's explained in the schema files, but good idea to explain it here.